arangodb-net-standard/DocumentApi/IDocumentApiClient.cs

```diff
@@ -20,12 +20,14 @@ public interface IDocumentApiClient
         /// <param name="query"></param>
         /// <param name="serializationOptions">The serialization options. When the value is null the
         /// the serialization options should be provided by the serializer, otherwise the given options should be used.</param>
+        /// <param name="headers">The <see cref="HeadDocumentHeader"/> values.</param>
         /// <returns></returns>
         Task<PostDocumentResponse<T>> PostDocumentAsync<T>(
            string collectionName,
            T document,
            PostDocumentsQuery query = null,
-           ApiClientSerializationOptions serializationOptions = null);
+           ApiClientSerializationOptions serializationOptions = null,
+           HeadDocumentHeader headers = null);
```

Comment:
Minor thing: we could rename `HeadDocumentHeader` to `DocumentHeader` or `DocumentHeaderProperties` to be in line with `CursorHeaderProperties`.